src/main/resources/application.properties

```diff
@@ -5,6 +5,10 @@ azure.application-insights.web.enable-W3C=true
 
 courts=Birmingham Immigration,Bromley,Caernarfon County,Caernarfon Crown,Caernarfon Magistrates,Cheltenham,City of London,Coventry Crown,Coventry Magistrates,Croydon Crown,Croydon Magistrates,Derby,Dudley,Hereford Crown,Hereford Magistrates,Inner London,Kidderminster,Kingston upon Thames,Leamington,Llandrindod Wells Justice Centre,Llandudno,Mold Crown,Mold County,Mold Magistrates,Prestatyn County,Redditch,Shrewsbury Crown,Shrewsbury Magistrates,South Derbyshire,Southwark,Telford,Thames,Walsall,Warwick,Welshpool County,Welshpool Magistrates,Westminster,Wimbledon,Wolverhampton Crown,Wolverhampton Magistrates,Wood Green,Woolwich,Worcester Crown,Worcester Magistrates,Wrexham County,Wrexham Magistrates
 
+courtIds=BROMMC,CRNRCT,CRNRCC,CRNHMC,CHELMC,CITYMC,CVNTCC,CVNTMC,CRYDCC,CRYDMC,DRBYCC,DUDLMC,HERFCC,HERFMC,INNRCC,KIDDMC,KNGTCC,LEAMMC,LLNWMC,LLDUMC,MOLDCC,MOLDCT,MOLDMC,PRSTMC,RDDTMC,SHRWCC,SHREMC,DRBYMC,STHWCC,TELFMC,THMSMC,WALSMC,WRWKCC,WLSHCT,WLSHMC,CRT034,WMBLMC,WLVRCC,WLVRMC,WDGRCC,WOOLCC,WRCSCC,WRCSMC,WREXCT,WREXMC
+
```

Comment:
for context, we return a small hardcoded list of courts based on roll out and these are used in a drop down in PSH and BVL. 

We're moving to use court ids and will do the enrichment with data from court register in the PSH

We're moving BVL over to use the full court list but at the moment we haven't got designs or planned out the work to move PSH so may need to maintain this list for sometime.